### ISSUE TITLE
rails g rapns created multiple migrations with the same timestamp + migration duplications

### ISSUE THREAD

**Issue by thrillcall** [ileitch/rapns#38]
Hi,

I just ran 'rails g rapns' and ended up with the following new files:

```
#   config/rapns/
#   db/migrate/20120214214103_create_rapns_feedback.rb
#   db/migrate/20120214214103_create_rapns_notifications.rb
#   db/migrate/20120214214104_add_alert_is_json_to_rapns_notifications.rb
```

However running rake db:migrate results in:

```
➜  git:(feature/rapns) ✗ rake db:migrate                                                                                      ruby-1.9.3-p0@wc-offers
rake aborted!
Multiple migrations have the version number 20120214214103

Tasks: TOP => db:migrate
(See full trace by running task with --trace)
```

I'm not sure if this is result of your generator or if my local Rails (3.2.1) freaked out.

Also, the last migration that was generated to 'add_alert_is_json_to_rapns_notifications' seems like it is simply adding a new column to the first migration table generated.  Could this not be rolled into the migration that does 'create_table :rapns_notifications'?

**Comment by ileitch** [comment]
I've just released 1.0.6 which should fix the issue with non-unique timestamps.

`add_alert_is_json_to_rapns_notifications` is a separate migration for users upgrading rapns. You should run `rails g rapns` each time to upgrade to check for new migrations and configuration changes.

**Comment by thrillcall** [comment]
Great!  Thanks.
